diff -uNr ng.orig/sys/sysv/spawn.c ng/sys/sysv/spawn.c
--- ng.orig/sys/sysv/spawn.c	Tue Jun 27 10:59:43 2000
+++ ng/sys/sysv/spawn.c	Thu Nov 23 11:47:49 2000
@@ -192,11 +192,11 @@
 	char *tmp;
 	static char tmpbuf[20];
 	int ostdin, ostdout, ostderr, in, out, s;
-	extern char *mktemp();
 
 	strcpy(tmpbuf, "/tmp/ngXXXXXX");
-	if ((tmp = mktemp(tmpbuf)) == NULL)
+	if (mkstemp(tmpbuf) == -1)
 		return NULL;
+	tmp = tmpbuf;
 	if ((in = open(input ? input : "/dev/null", 0)) < 0)
 		return NULL;
 	if ((out = creat(tmp, S_IREAD | S_IWRITE)) < 0) {
