diff -ur ng-1.4.4.orig/sys/sysv/sysdef.h ng-1.4.4/sys/sysv/sysdef.h
--- ng-1.4.4.orig/sys/sysv/sysdef.h	2003-02-23 23:32:58.000000000 +0900
+++ ng-1.4.4/sys/sysv/sysdef.h	2005-06-14 16:50:56.000000000 +0900
@@ -52,11 +52,7 @@
 #define MALLOCROUND(m)	(m+=7,m&=~7)	/* round up to 8 byte boundry	*/
 
 #define	fncmp		strcmp		/* file name comparison		*/
-#define bcopy(s,d,n)	memcpy(d,s,n)	/* memory-to-memory copy	*/
-#define bzero(s,n)	memset(s,0,n)	/* 91.01.21  Add by H.Kaneko.	*/
-#define bcmp(s,d,n)	memcmp(s,d,n)	/* 91.02.04  Add by Y.Nimura.	*/
-					/* 91.02.04  strncmp -> memcmp	*/
-					/*		by S.Yoshida	*/
+
 #ifndef SVR2		/* SVID2(==SVR3) has rmdir. 00.04.03 by amura	*/
 #define	unlinkdir(fn)	rmdir(fn)	/* unlink directory		*/
 #endif
